service/etc/agama.yaml

```diff
@@ -20,6 +20,88 @@ web:
   ssl_cert: null
   ssl_key: null
 
+ALP-Dolomite:
+  software:
+    installation_repositories:
+      - url: https://updates.suse.com/SUSE/Products/ALP-Dolomite/1.0/x86_64/product/
+        archs: x86_64
+      - url: https://updates.suse.com/SUSE/Products/ALP-Dolomite/1.0/aarch64/product/
+        archs: aarch64
+      - url: https://updates.suse.com/SUSE/Products/ALP-Dolomite/1.0/s390x/product/
+        archs: s390
+      - url: https://updates.suse.com/SUSE/Products/ALP-Dolomite/1.0/ppc64le/product/
+        archs: ppc
+
+    mandatory_patterns:
+      - alp_base_zypper
+      - alp_cockpit
+      - alp_hardware
+    optional_patterns: null # no optional pattern shared
+    mandatory_packages:
+      - package: device-mapper # Apparently needed if devices at /dev/mapper are used at boot (eg. FDE)
+      - package: fde-tools # Needed for FDE with TPM, hardcoded here temporarily
+        archs: aarch64, x86_64
+      - package: libtss2-tcti-device0 # Same than fde-tools
```

Comment:
I do not get comment, what means "Same than"? I would use "Same case as fde-tools". But then why fde tools are arch based and this not?

Reply:
These packages where orininally added here https://github.com/openSUSE/agama/pull/485 for ALP-Bedrock, and the architecture filter was added here https://github.com/openSUSE/agama/pull/486. Maybe @teclator and @imobachgs remember some details.

Reply:
For now, let's remove this misleading comment. All these about FDE, etc should be revisit, but it is out of scope of this PR.